src/sentry/runner/settings.py

```diff
@@ -1,4 +1,5 @@
 import os
+import warnings
```

Comment:
Pulled to module level for https://github.com/getsentry/sentry/pull/26796/files#diff-365515e67962b3e3e432dbfa855776b1159e5405c11ea89f8962f73cc496ea1cR92.